packages/xod-client/src/core/containers/App.jsx

```diff
@@ -91,6 +105,33 @@ export default class App extends React.Component {
     document.addEventListener('copy', this.props.actions.copyEntities);
     document.addEventListener('paste', this.props.actions.pasteEntities);
     this.props.actions.fetchGrant(/* startup */ true);
+
+    // TODO: Replace with ref and `React.createRef`
+    //       after updating React >16.3
```

Comment:
Just curious. Why React >16.**3** is needed for `createRef`?

Reply:
![image](https://user-images.githubusercontent.com/1897530/98365421-74d5d300-2043-11eb-9de5-7202b0505fdf.png)
Screenshot from the React documentation. However, we can use a "callback ref" as usually 🤔